COMPATIBILITY.md

```diff
@@ -87,7 +91,7 @@ Entries list the class name serving as the entry point to the relevant functiona
 - [X] `ec2`: known to work as is
```

Comment:
Could say something like “known to work at least in version 1.23”, though that could be misread as implying that older or newer versions do not work, when really what we want to say is that we (of course) tested a specific version, which worked, and there is no reason to believe that other versions do not work as well.